hrui/js/src/contact.js

```diff
@@ -22,6 +22,16 @@
       if (typeof data.roles !== 'undefined')  {
         updateContactHeaderRolesDetails(data.roles);
       }
+    })
+
+    // Actions menu
+    .on('click', function(e) {
```

Comment:
We should put this code as part of `uk.co.compucorp.civicrm.hrcontactactionsmenu/` extension's JS files.